File: test_scripts/hello.js
#!/usr/bin/env node
/**
 * 测试脚本 - Node.js Hello World
 */

const os = require('os');

console.log('='.repeat(50));
console.log('Node.js 脚本执行测试');
console.log('='.repeat(50));

// 打印基本信息
console.log(`执行时间: ${new Date().toLocaleString('zh-CN')}`);
console.log(`Node.js 版本: ${process.version}`);
console.log(`平台: ${os.platform()}`);
console.log(`架构: ${os.arch()}`);

// 处理命令行参数
const args = process.argv.slice(2);
if (args.length > 0) {
    console.log(`\n收到 ${args.length} 个参数:`);
    args.forEach((arg, index) => {
        console.log(`  参数 ${index + 1}: ${arg}`);
    });
} else {
    console.log('\n未收到命令行参数');
}

// 生成示例输出
const result = {
    status: 'success',
    message: 'Node.js 脚本执行成功',
    timestamp: new Date().toISOString(),
    args: args,
    environment: {
        nodeVersion: process.version,
        platform: os.platform(),
        arch: os.arch()
    }
};

console.log('\n执行结果:');
console.log(JSON.stringify(result, null, 2));

console.log('\n脚本执行完成！');

File: test_scripts/hello.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
测试脚本 - Python Hello World
"""

import sys
import json
from datetime import datetime

def main():
    print("=" * 50)
    print("Python 脚本执行测试")
    print("=" * 50)

    # 打印基本信息
    print(f"执行时间: {datetime.now().strftime('%Y-%m-%d %H:%M:%S')}")
    print(f"Python 版本: {sys.version}")
    print(f"命令行参数: {sys.argv[1:]}")

    # 处理命令行参数
    if len(sys.argv) > 1:
        print(f"\n收到 {len(sys.argv) - 1} 个参数:")
        for i, arg in enumerate(sys.argv[1:], 1):
            print(f"  参数 {i}: {arg}")

    # 生成示例输出
    result = {
        "status": "success",
        "message": "Python 脚本执行成功",
        "timestamp": datetime.now().isoformat(),
        "args": sys.argv[1:] if len(sys.argv) > 1 else []
    }

    print("\n执行结果:")
    print(json.dumps(result, ensure_ascii=False, indent=2))

    print("\n脚本执行完成！")

if __name__ == "__main__":
    main()
